### PR TITLE
Include MariaDB in doc "Generic MySQL backend"

### PR DESCRIPTION
**Pull request by robertsilen**
### Short description
Added MariaDB to doc "Generic MySQL backend", as discussed in [github.com/PowerDNS/pdns/discussions/14538](https://github.com/PowerDNS/pdns/discussions/14538). 

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/backends/generic-mysql.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Generic MySQL backend
+Generic MySQL/MariaDB  backend
 =====================
```
Comment (grooverdan) [inline docs/backends/generic-mysql.rst:2]: nit: underscores match title length. Same with other header.

Comment (robertsilen) [reply docs/backends/generic-mysql.rst]: Fixed

**Comment by robertsilen** [comment]
Hello @rgacogne , sorry for nagging, but does this seem ok now?

**Comment by Habbie** [comment]
> Hello @rgacogne , sorry for nagging, but does this seem ok now?

I'll have a look at this soon!

**Review by Habbie — APPROVED** [review]
LGTM, although we could consider updating the table entry in `docs/backends/index.rst` too

**Comment by robertsilen** [comment]
> LGTM, although we could consider updating the table entry in `docs/backends/index.rst` too

Add an own row for MariaDB on the [Backends doc page](https://doc.powerdns.com/authoritative/backends/) (in [github](https://github.com/PowerDNS/pdns/blob/master/docs/backends/index.rst))? I would, but I don't know enough to fill it out for MariaDB. Suggestions?

**Comment by Habbie** [comment]
> Add an own row for MariaDB on the [Backends doc page](https://doc.powerdns.com/authoritative/backends/) (in [github](https://github.com/PowerDNS/pdns/blob/master/docs/backends/index.rst))? I would, but I don't know enough to fill it out for MariaDB. Suggestions?

no, just extend the name of the existing one. it's not a separate backend.

**Comment by robertsilen** [comment]
> > Add an own row for MariaDB on the [Backends doc page](https://doc.powerdns.com/authoritative/backends/) (in [github](https://github.com/PowerDNS/pdns/blob/master/docs/backends/index.rst))? I would, but I don't know enough to fill it out for MariaDB. Suggestions?
> 
> no, just extend the name of the existing one. it's not a separate backend.

Sure, not sure if I could have added it to this PR, but it became a separate one now: https://github.com/PowerDNS/pdns/pull/14546

**Comment by Habbie** [comment]
> Sure, not sure if I could have added it to this PR, but it became a separate one now: #14546

From the commandline, you certainly could have. I don't know if it's easy in the Github editor. Either way works :) Will merge the other one when CI is green, thanks!

**Pull request merged**